server/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -132,7 +133,7 @@ public static MultiSearchRequest parseRequest(
         boolean allowExplicitIndex,
         SearchUsageHolder searchUsageHolder,
         Predicate<NodeFeature> clusterSupportsFeature,
-        TriFunction<String, Object, SearchRequest, Boolean> extraParamParser
```

Comment:
We're changing how additional params are parsed. Instead of doing the actual parsing for each key, we use the RequestParams to check for additional params that needs to be taken into account